recipes/rapifilt/meta.yaml

```diff
@@ -19,6 +18,8 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - zlib
+    - htslib 
+    - boost
```

Comment:
This is a python library, you just want boost-cpp

Reply:
done